gen-python/opencensus/proto/agent/trace/v1/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
This line is intended for [namespace packages](https://packaging.python.org/guides/packaging-namespace-packages/).
If there is no plan to make `opencensus/proto/agent/trace/v1 a namespace` (for example, having multiple packages sharing the `opencensus.proto.agent.trace.v1 namespace`), we can remove this line.

Reply:
For v1 we won't have multiple packages. However we may have multiple packages (versions) share the upper level package. For example `opencensus/proto/agent/trace/v1` and `opencensus/proto/agent/trace/v2` will share `opencensus/proto/agent/trace` - kindly correct me if I'm wrong.

Removed all paths from */v1 for now.

Reply:
Yes, this (having v1 and v2) is what I was thinking, too :)

Reply:
As it is now I don't think you need the incantation for namespace packages. If we release v1 and v2 as separate distributions, each with a dependency on a separate common distribution, then we'll need it.

What's our support story for v1 protos? If we're not updating them, could we release them as an earlier version of this library and drop support for later versions?

Reply:
> If we release v1 and v2 as separate distributions, each with a dependency on a separate common distribution, then we'll need it.

I think this is what we'll end up doing. In the future we'll have packages like `opencensus/proto/common/v1` and `opencensus/proto/common/v2`. All the */v1 packages will depend on common/v1 and all v2 packages depend on common/v2.

> What's our support story for v1 protos? If we're not updating them, could we release them as an earlier version of this library and drop support for later versions?

I don't have a clear answer to this but my idea is when v2 is out, we'll keep v1 protos there and continue releasing them with the same version number, though v1 protos won't be forward-compatible. (v2 protos are unlikely to be backwards-compatible either.)

Reply:
That makes sense. You may just want to add the namespace package logic when you add a second `setup.py` then.